modules/gltf/src/lib/gltf-scenegraph.js

```diff
@@ -418,4 +464,44 @@ export default class GLTFScenegraph {
       }
     }
   }
+
+  // Add attributes to buffers and create `attributes` object which is part of `mesh`
```

Comment:
Is this new, or just moved? It looks familiar.

Reply:
It's new. `addMesh` called undefined method `_addAttributes`